applications/GeoMechanicsApplication/tests/1D-Consolidation_all_stages/README.md

```diff
@@ -4,13 +4,13 @@ This test verifies that in a column of triangular plane strain elements, consoli
 
 ## Setup
 
-This test consists of 78 6-noded elements (using the SmallStrainUPwDiffOrderElement2D6N class) of $b x h = 0.1 x 1.0 \mathrm{[m]}$. A
+This test consists of 78 6-noded elements (using the SmallStrainUPwDiffOrderElement2D6N class) of $b \times h = 0.1 \times 1.0 \mathrm{[m]}$. A
 schematic representation can be found in the figure below:
 
 ![MeshStructure](MeshStructure.svg)
 
 All nodes on the sides have fixed horizontal displacements. The bottom nodes have fixed displacement in both the horizontal and the vertical
-direction. At the top of the column a vertical compressive line load with a magnitude of 1.0 N/m is applied. This load is kept constant during the whole analysis time. Note that gravity is absent in this computation, the top load generates the excess pore pressure. In the first stage no Dirichlet boundaries for the water pressure D.O.F. are given, such that all water is contained within the column. I n all later stages, the water pressure (p) on the top of the column is specified to be 0.0 [Pa]. This creates an outflow boundary at the top.
+direction. At the top of the column a vertical compressive line load with a magnitude of $q=1.0 \mathrm{[N/m]}$ is applied. This load is kept constant during the whole analysis time. Note that gravity is absent in this computation, the top load generates the excess pore pressure. In the first stage no Dirichlet boundaries for the water pressure D.O.F. are given, such that all water is contained within the column. In all later stages, the water pressure (p) on the top of the column is specified to be 0.0 [Pa]. This creates an outflow boundary at the top.
```

Comment:
Thank you for fixing my own inconsistencies.

Reply:
Welcome, You also fix my problems.